### ISSUE TITLE
Ensure all navigation-related images have helpful "alt" text

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#258]
_Part of epic: #257_

There are some places in the website where we use images as part of a subsection navigation arrangement. For example, on the Guide portal pages like http://isaw.nyu.edu/guide:

![screen shot 2018-04-25 at 4 31 01 pm](https://user-images.githubusercontent.com/263285/39273962-11fcf3b6-48a6-11e8-8fe9-42e5d37c19b1.png)

On the page in question there are "alt" attributes on the ```<img>``` elements, but they simply replicate the filename. For example:

```html
<td>
    <a 
        title="" 
        href="http://isaw.nyu.edu/guide/portals/for-students" 
        class="internal-link" 
        target="_self">
        <img 
            src="http://isaw.nyu.edu/guide/images/student2.jpg" 
            alt="Student-2.jpg" 
            class="image-inline" 
            title="Student-2.jpg" />
    </a>
</td>
```

The content of the "alt" attributes should be descriptive as useful. Note also that the title attribute is not being used well either. Note further that these images have labeling embedded in the raster image itself that is not replicated in text anywhere in the HTML supporting the image or the link.

We need to find all instances of this on the website and fix them once and for all.

**Comment by paregorios** [comment]
Since this particular issue is about images that we use in navigation-like contexts, this needs to be fixed by the 1 August 2018 milestone.

**Comment by paregorios** [comment]
yep, added "manual content work" tag
